lib/util/common.ts

```diff
@@ -59,3 +61,19 @@ export function detectPlatform(
 
   return null;
 }
+
+export function parseJsonWithFallback(content: string, context: string): any {
```

Comment:
Can we return `unknown` instead, so any consumer knows this is an unsafe object and needs validation?